x-pack/plugins/lens/public/visualizations/xy/xy_config_panel/axis_settings_popover.test.tsx

```diff
@@ -286,5 +271,175 @@ describe('Axes Settings', () => {
       expect(lower.prop('value')).toEqual(123);
       expect(upper.prop('value')).toEqual(456);
     });
+
+    describe('Custom bounds', () => {
```

Comment:
Same as in the other file, it would be nice if we could have these tests written in rtl, even if you're not planning on re-writing the whole file. If you decide to do so, a useful file could be this one to look at: `x-pack/plugins/lens/public/visualizations/datatable/components/toolbar.test.tsx` where there are some interactions with button group too. We use there `import { getSelectedButtonInGroup } from '@kbn/test-eui-helpers';` that might be useful.

Reply:
Thanks for this comment, RTL is so simple, a little different paradigm than enzyme but way better!

Rewrote tests to RTL in a2a5cb10b71681314c792b1ebffe6af9a862005a